fedimint-testing/src/fixtures.rs

```diff
@@ -187,7 +187,9 @@ impl Fixtures {
         let listen: SocketAddr = "127.0.0.1:9000".parse().unwrap();
         let address: SafeUrl = format!("http://{listen}").parse().unwrap();
 
-        let ln_client: Arc<dyn ILnRpcClient> = lightning_builder.build().await.into();
+        // TODO: Pass in the test runtime rather than spawning a new one.
+        let runtime = Arc::new(tokio::runtime::Runtime::new().unwrap());
```

Comment:
not sure if this is going to work, you are inside a `Runtime` already (because you need to run an async fn) so if you create a runtime here you will have some panic later that tells you that there are two runtime running

Reply:
Hmmm strange. Something's clearly broken since CI is currently failing, but `ldk-node::Node::start()`, which we were calling previously, [creates its own runtime](https://docs.rs/ldk-node/0.4.2/src/ldk_node/lib.rs.html#212) and it didn't cause any CI issues.

I'll keep digging...

Reply:
The line I added that fixed this issue was this:

`fedimint_core::runtime::block_in_place(|| drop(runtime));`